chart/compass/templates/gateway-auditlog-integration-test.yaml

```diff
@@ -52,15 +52,13 @@ spec:
                   key: password
             - name: DEFAULT_TENANT
               value: "{{.Values.global.defaultTenant}}"
-            - name: DOMAIN
```

Comment:
Why was this env removed? I think it's still used by dex config.

Reply:
I'm sorry but the only usage I found for this env was [here](https://github.com/kyma-incubator/compass/pull/1315/files#diff-675e1eb998cbe308ed172355da377222L15) and this variable was unused too.

Reply:
Ok, found the usage